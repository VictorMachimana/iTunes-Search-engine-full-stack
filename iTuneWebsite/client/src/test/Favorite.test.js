import React from "react";
import Music from "../components/music";
import renderer from "react-test-renderer";

test("renders correctly", () => {
  const tree = renderer.create(<Music />).toJSON();

  expect(tree).toMatchSnapshot();
});
