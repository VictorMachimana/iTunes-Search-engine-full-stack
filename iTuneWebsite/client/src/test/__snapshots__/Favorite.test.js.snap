// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="favorite"
  >
    <table
      className="table table-bordered table-dark"
    >
      <thead>
        <tr>
          <th>
            #Favorite List
          </th>
          <th>
            <i
              aria-hidden="true"
              class="fa fa-music"
            />
          </th>
        </tr>
      </thead>
    </table>
    <table
      className="table table-bordered"
    >
      <tbody>
        <tr
          className="row-container"
        />
      </tbody>
    </table>
  </div>
  <h2
    className="heading"
  >
    Search For Music Below
  </h2>
  <label
    className="search-label"
    htmlFor="search-input"
  >
    <input
      id="search-input"
      name="query"
      onChange={[Function]}
      placeholder="Search..."
      type="text"
      value=""
    />
    <i
      aria-hidden="true"
      className="fa fa-search search-icon"
    />
  </label>
  
  <img
    alt="loader"
    className="search-loading hide"
    src="loader.gif"
  />
  <div
    className="nav-link-container"
  >
    <a
      className="nav-link hide 
          "
      href="/"
      onClick={[Function]}
    >
      Prev
    </a>
    <a
      className="nav-link hide
          "
      href="/"
      onClick={[Function]}
    >
      Next
    </a>
  </div>
  <div
    className="nav-link-container"
  >
    <a
      className="nav-link hide 
          "
      href="/"
      onClick={[Function]}
    >
      Prev
    </a>
    <a
      className="nav-link hide
          "
      href="/"
      onClick={[Function]}
    >
      Next
    </a>
  </div>
</div>
`;
